src/renderer/components/Button.tsx

```diff
@@ -1,86 +1,41 @@
-// @flow
-
-import React, { PureComponent } from "react";
-import noop from "lodash/noop";
-import { Button as BaseButton } from "@ledgerhq/react-ui";
+import React from "react";
+import { Button as BaseButton, InvertTheme } from "@ledgerhq/react-ui";
+import { ButtonProps as BaseButtonProps } from "@ledgerhq/react-ui/components/cta/Button";
 import { track } from "~/renderer/analytics/segment";
-import { isGlobalTabEnabled } from "~/config/global-tab";
 
-export const Base: any = BaseButton;
+export const Base: typeof BaseButton = BaseButton;
```

Comment:
Is this type annotation even necessary? Doesn't TypeScript infer it?

Reply:
> Doesn't TypeScript infer it?

Unfortunately nope :/.

```txt
The inferred type of 'Base' cannot be named without a reference to '@ledgerhq/react-ui/node_modules/@types/react'.
This is likely not portable. A type annotation is necessary.
```